src/tools/kata-ctl/src/arch/aarch64/mod.rs

```diff
@@ -8,8 +8,18 @@ pub use arch_specific::*;
 
 mod arch_specific {
     use anyhow::Result;
+    use std::path::Path;
+
+    const KVM_DEV: &str = "/dev/kvm";
 
     pub fn check() -> Result<()> {
-        unimplemented!("Check not implemented in aarch64")
+        println!("INFO: check: aarch64");
+        if Path::new(KVM_DEV).exists() {
+            println!("Kata Containers can run on this host\n");
```

Comment:
nit: May be add a hint that, with the VMMs which support aarch64.

Reply:
@Ji-Xinyou Line 16 prints aarch64, so I think we can just merge it.